webdriver/pom.xml

```diff
@@ -37,7 +37,7 @@
     </distributionManagement>
 
     <properties>
-        <selenium.version>2.47.0</selenium.version>
+        <selenium.version>2.49.1</selenium.version>
```

Comment:
This change is not mentioned and should not be here. Bumping up Selenium dependency is whole different topic.